src/Payments.sol

```diff
@@ -934,18 +923,6 @@ contract Payments is
         Rail storage rail,
         Account storage payer
     ) internal {
-        // Get operator approval to reduce usage
-        OperatorApproval storage approval = operatorApprovals[rail.token][
-            rail.from
-        ][rail.operator];
-
```

Comment:
This now happens in the "settleAccountLockupBeforeAndAfterForRail" modifier that is called on "settleTerminatedRailWithoutArbitration" and "settleRail"